ProjectLighthouse.Localization/Moderation.resx

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<root>
+    <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
+        <xsd:element name="root" msdata:IsDataSet="true">
+            
+        </xsd:element>
+    </xsd:schema>
+    <resheader name="resmimetype">
+        <value>text/microsoft-resx</value>
+    </resheader>
+    <resheader name="version">
+        <value>1.3</value>
+    </resheader>
+    <resheader name="reader">
+        <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+    </resheader>
+    <resheader name="writer">
+        <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+    </resheader>
+    <data name="banned_heading" xml:space="preserve">
+        <value>Account Banned</value>
+    </data>
+    <data name="banned_explain" xml:space="preserve">
+        <value>Your account has been banned due to inappropriate, offensive, or violent behavior.</value>
+    </data>
+    <data name="case_created" xml:space="preserve">
+        <value>Case Created</value>
+    </data>
+    <data name="case_expires" xml:space="preserve">
+        <value>Case Expires</value>
+    </data>
+    <data name="ban_reason" xml:space="preserve">
+        <value>Ban Reason</value>
+    </data>
+    <data name="appeal_instructions" xml:space="preserve">
+        <value>If you believe your ban was unjust, please contact an instance moderator.</value>
```

Comment:
Unless we are also going to implement an appeals system at some point it seems better to exclude the hardcoded appeal. You could instead have moderators include instructions for appeal in the ban message itself.

Reply:
I'd have to check the current ban string but I believe it mentions something of the same aspect.